### PR TITLE
remove replace

### PR DESCRIPTION
**Pull request by cpanato**
Looks like this is breaking new PRs in this repo, like https://github.com/mattermost/mattermost-plugin-github/pull/147

### CODE REVIEW

**Comment by cpanato** [comment]
@levb are you using go 1.13?

**Comment by levb** [comment]
Oh thanks for pointing it out, I switched back to 1.12.10 recently to help someone out, and forgot to switch back.

File: go.mod [hunk 1]
```diff
@@ -24,6 +24,4 @@ require (
 
 replace (
```
Comment (hanzei) [inline go.mod]: My local go commands replace this with 
```
replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Would you please run `go mod tidy` once?

Comment (cpanato) [reply go.mod]: done

**Pull request merged**